### PR TITLE
Welcome Email using Cloud Functions 

### PR DESCRIPTION
**Pull request by christyjacob4**
Cloud function demo in Nodejs leveraging the MailGun Email API to send welcome emails

### CODE REVIEW

File: nodejs/welcome-email/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+* Upload the tarfile to your Appwrite Console and use the following entrypoint command
+
+```bash
+node index.js
+```
+
+## 🎯 Trigger
+
+Head over to your function in the Appwrite console and under the Settings Tab, enable the `users.create` event.
```
Comment (eldadfux) [inline nodejs/welcome-email/README.md:44]: You need both user.create and account.create. They have the same payload, but triggered by different endpoints

File: nodejs/welcome-email/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+const mg = mailgun(emailConfig);
+
+// Get the name and email of the newly created user from Appwrite's environment variable
+const payload = JSON.parse(process.env.APPWRITE_FUNCTION_EVENT_PAYLOAD)
+const name = payload['name']
+const email = payload['email']
+
+// Create your email 
+const data = {
+	from: 'Welcome to Appwrite <welcome@appwrite.io>',
```
Comment (eldadfux) [inline nodejs/welcome-email/index.js:18]: "Appwrite" might be confusing, what about "My Awesome New App"?

**Review by eldadfux — CHANGES_REQUESTED** [review]
Minor comments, great work, much faster too!

File: nodejs/welcome-email/index.js (hunk 2 above)
Comment (christyjacob4) [reply nodejs/welcome-email/index.js:18]: Yea I agree. Will make those changes

File: nodejs/welcome-email/README.md (hunk 1 above)
Comment (christyjacob4) [reply nodejs/welcome-email/README.md:44]: But when I was using the console to create a new user, I was able to trigger it with the `users.create` event 🤔

Comment (eldadfux) [reply nodejs/welcome-email/README.md:44]: yeh 100%, but for the function to be usable to end devs you would need to use both events.

Comment (christyjacob4) [reply nodejs/welcome-email/README.md:44]: okay understood

**Review by eldadfux — APPROVED** [review]

**Comment by eldadfux** [comment]
🔥

**Pull request merged**